src/app/containers/ChartbeatAnalytics/amp/index.jsx

```diff
@@ -10,7 +11,7 @@ const JsonInlinedScript = data => (
   <script
     type="application/json"
     /* eslint-disable-next-line react/no-danger */
-    dangerouslySetInnerHTML={{ __html: JSON.stringify(data) }}
+    dangerouslySetInnerHTML={{ __html: xssJsonStringify(data) }}
```

Comment:
I'm not sure you need to use `dangerouslySetInnerHTML` here since. I think you can just do:

```jsx
<script type="application/json">{JSON.stringify(data)}</script>
```

An example of this can be found here - https://github.com/bbc/simorgh/blob/latest/src/app/components/LinkedData/index.jsx#L81 ignore the `/* eslint-disable react/no-danger */` on the line above this is also not needed.